helm/chart/router/templates/deployment.yaml

```diff
@@ -72,8 +72,10 @@ spec:
             - /app/configuration.yaml
             {{- end }}
           {{- end }}
-          {{- if or .Values.managedFederation.apiKey .Values.managedFederation.existingSecret .Values.managedFederation.graphRef .Values.extraEnvVars }}
           env:
+            - name: APOLLO_OFFICIAL_ROUTER_HELM_CHART
```

Comment:
The naming convention is:
```
APOLLO_ROUTER_<blah>
```

Is this deliberately not following the convention for some reason?

Reply:
No that's just me not knowing the convention, I'll get that fixed up :)